### DISCUSSION TITLE
day1- sudo ubuntu-drivers autoinstall를 입력하면 오류가 뜹니다.

### DISCUSSION THREAD

**Discussion by Jeoungji** [Road-Balance/gcamp_ros2_basic#32] [Q&A] [answered]
gazebo에서 화면이 
sudo ubuntu-drivers autoinstall를 입력하면, 다음과 같은 오류가 뜹니다.
![오류](https://user-images.githubusercontent.com/109124955/178484229-1c4cc493-d453-43d3-bb90-de722342a32c.PNG)

델 g3 15 2018년 i5 1050ti모델입니다.
![오류1](https://user-images.githubusercontent.com/109124955/178484243-d0dc1b5b-1be6-4e48-b0ae-f13bcf1abe95.PNG)
위와 같이 드라이버는 설치를 하였습니다.

**Comment by kimsooyoung** [comment, marked as answer]
https://velog.io/@jaehyeong/WSL2-%EC%B4%88%EA%B0%84%EB%8B%A8-%EC%84%A4%EC%B9%98-%EB%B0%8F-CUDAGPU-%EC%84%A4%EC%A0%95-%EB%B0%A9%EB%B2%95

ㄴ 위 블로그 링크를 참고하셔서 다시 시도를 부탁드립니다.

**Reply by Jeoungji** [reply]
![오류5](https://user-images.githubusercontent.com/109124955/178561202-764a9e30-7040-4cd5-96c4-fd35021a5a41.PNG)
위와 같이 블로그에 내용을 따라한 뒤 sudo ubuntu-drivers autoinstall을 다시 실행해보았지만
![오류6](https://user-images.githubusercontent.com/109124955/178561457-0320df56-9f3f-4bb4-bbd7-0e28898b0f12.PNG)
똑같은 오류가 뜹니다.
ubuntu-drivers devices를 실행해보아도 위와 같은 오류가 떴습니다.
![dhfb4](https://user-images.githubusercontent.com/109124955/178561973-19f1df89-a8eb-409d-a5c7-8d4b8f7abd13.PNG)
gazebo에서는 위와 같이 그림자가 나오지 않습니다.

**Reply by kimsooyoung** [reply]
개인별 모든 PC 상황을 확실히 알기는 쉽지 않습니다. 

해당 문제는 구글 검색으로 해결하시고 예를 들어 제가 찾은 결과는 다음과 같습니다. 

https://www.nemotos.net/?p=5061

앞으로 문제가 발생하면 구글 검색을 통해 우선 조회하시고, 이 섹션에는 ROS 관련된 부분에서만 커멘트를 남겨주세요


화이팅입니다.

**Reply by Jeoungji** [reply]
넵 감사합니다
